### ISSUE TITLE
Routes with more than 14 Stations get approved

### ISSUE THREAD

**Issue by quchasz** [edk-software/edk-route-verifier#145]
As visible on the examples below - routes with 15 and 16 Stations get approved.
![Screen Shot 2020-02-21 at 18 27 27](https://user-images.githubusercontent.com/47346557/75057014-5ee5a980-54d8-11ea-87d0-98f281b2e8da.png)
![Screen Shot 2020-02-21 at 18 30 19](https://user-images.githubusercontent.com/47346557/75057031-66a54e00-54d8-11ea-88d5-b7a5751911c5.png)

Example kml files attached:
[test_stacja_15 i 16.zip](https://github.com/edk-software/edk-route-verifier/files/4237291/test_stacja_15.i.16.zip)

**Comment by qooban** [comment]
Such points (named "Stacja XV", "Stacja XVI") are ignored by verifier at the moment. The same as we allow to have points different than stations at the moment - eg. "Początek trasy" or "EDK - something".

Until we have very clear rules about naming I would prefer not to change that. You can have point named "EDK - Benedict XVI" which should not be blocked with current state of requirements.

Btw I opted some time ago to allow only points with latin or roman numbers (ranged from 1 to 14): "1", "2", "I", "II", "X", "XII". That would simplify verification a lot.

**Comment by quchasz** [comment]
I agree - limit to 14 points with latin or roman numbers only "1", "2"... or "I", "II",... is fine - however doing it now is too risky due to the numer or routes already approved, and route requirements already introduced to area leaders.

However for the next edition we should do it.
